low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -285,3 +309,142 @@ def get_fans_info(self):
 
             data.append(fan_dict)
         return data
+
+    def initialize_systemd(self):
```

Comment:
We can maintain these generic functions in respective classes, like 
low-level/framework/platforms/server/software.py
And have a class under that as SwService(), OS(), Build(), etc

Reply:
Sure @cdeshmukh, I will move these functions to corresponding classes.

Reply:
Moved functions to `BuildInfo` and `Service` classes under `low-level/framework/platforms/server/software.py`.